ncm-authconfig/src/main/pan/components/authconfig/schema.pan

```diff
@@ -246,15 +244,23 @@ type authconfig_method_type = {
 
 type hash_string = string with match(SELF, "^(descrypt|md5|sha256|sha512)$");
 
-type component_authconfig_type = {
+type authconfig_component = {
     include structure_component
+    @{When set to true, no actual configuration will change. Default: false.}
     "safemode" : boolean = false
     "passalgorithm" : hash_string = "md5"
+    @{Enable the use of shadow password files.}
     "useshadow" ? boolean
+    @{Enable or disable nscd operation.}
     "usecache" ? boolean
     "enableforcelegacy" : boolean = false
     "startstop" ? boolean
+    @{Enable the use of MD5 hashed password.}
     "usemd5" : boolean
+    @{dict of authentication methods to enable. Supported
+    methods are: files, ldap, nis, krb5, smb, hesiod, afs, nslcd and sssd.
+    Note that "afs" is only supported on the CERN-modified version of
```

Comment:
was moving pod to schema, not really checking everything. i'll recheck the code and remove any references if it is not in the current code.